src/WpfMath/TexFontInfo.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;

```

Comment:
What's going here? Seems like an identical line, hrmm.

Reply:
I have no idea..

Reply:
This is what git diff says.

@@ -1,4 +1,4 @@
-<EF><BB><BF>using System;
+using System;^M

I must admit not exactly sure what this means or how i fix it.

Reply:
Ah, it's a Windows-style line ending (`^M` is a carriage return). Maybe this will help: https://superuser.com/questions/194668/grep-to-find-files-that-contain-m-windows-carriage-return

Reply:
Yeah i figured it out, the file had been saved as UTF-8 instead of UTF-8-BOM. It should be fixed now.

Reply:
Regarding BOM: we've [discussed these a while ago](https://github.com/ForNeVeR/wpf-math/pull/71#discussion_r105537740), but nobody was particularly interested, so I single-handedly decided to let the BOM stay as it is. Probably we'll need a small research regarding current behavior of compilers in case we want to remove it. Current "consensus" is that we use BOM (because it helps .NET compilers to work with Unicode in sources), and I'd prefer everyone (incl. myself) to not change that without further discussion.